contents/tutorials/churn-rate.mdx

```diff
@@ -45,7 +60,12 @@ Another way to visualize churn is through lifecycle charts. Lifecycle charts sho
 
 The dormant users are the ones who churned. Comparing them with the ones who did the event (new, returning, or resurrected) can give you the churn rate for that event or action.
 
-![Lifecycle chart](../images/tutorials/churn-rate/lifecycle-chart.png)
+<ProductScreenshot
```

Comment:
The "group by" here should match the image and text above. If you change text in previous section to "last 10 days" it works fine